demo/i18n.html

```diff
@@ -72,9 +72,9 @@
         "Autocomplete suggestions": "Предложения автозаполнения",
                 
         "Cursor at row $0": "",
-        "Unfold rows $0 to $1": "",
+        "Code folding, rows $0 through $1": "",
+        "Code folding, row $0": "",
         "Unfold code": "",
-        "Fold at row $0": "",
```

Comment:
if someone was using the old translation keys, what is going to happen? can we at least show a warning? can we support both by deprecating the old one?

Reply:
added a warning when a Ace tries to replace a string using the `nls` function but can't find a translation in the provided messages.

Reply:
Does this mean this is a breaking change? If so, can we avoid it or is it intentional?